build-configs/BuildConfigType.ts

```diff
@@ -51,7 +51,7 @@ export type CommonBuildConfigType = {
   hostName: string
   // Hostnames from which resources are automatically downloaded for offline usage.
   allowedHostNames: Array<string>
-  allowedIframeSources: string[]
+  whiteListedIframeSources: string[]
```

Comment:
I think we should avoid this term if possible bc of discrimination, why did you change it from `allowed`?